src/events/http/HttpServer.js

```diff
@@ -412,6 +412,39 @@ export default class HttpServer {
     return authStrategyName
   }
 
+  _setAuthorizationStrategy(endpoint, functionKey, method, path) {
+    /*
+     *  The authentication strategy can be provided outside of this project
+     *  by injecting the provider through a custom variable in the serverless.yml.
+     *
+     *  see the example in the tests for more details
+     *    /tests/integration/custom-authentication
+     */
+    const customizations = this.#serverless.service.custom
+    if (
+      customizations &&
+      customizations.offline?.customAuthenticationProvider
+    ) {
+      // eslint-disable-next-line import/no-dynamic-require, global-require
+      const provider = require(customizations.offline
```

Comment:
@medikoo -  Based on your earlier feedback of moving this away from a serverless plugin, let me know if you see a better way to inject this.  My desire would be that in serverless.yml you'd be able to add just a regular import statement in
`custom:
    offline:
        customAuthenticationProvider: 
`
this makes for an inline 'require', unless you know of a better way?